Formula/s/simg2img.rb

```diff
@@ -41,3 +41,17 @@ def install
     system "diff", "512k-zeros.img", "new-512k-zeros.img"
   end
 end
+
```

Comment:
Add this lines after the uses_from_macos,

```
  # Fix for arm from a fork
  patch do
    url "https://github.com/anestisb/android-simg2img/commit/931df9dd83e7feea11197402c5b4e7ad489f4abf.patch?full_index=1"
    sha256 "97f7e1256e9bc0fcfb4e1714ac503c7d5a4901a1685fe86307222d0f67ae5898"
  end

```

We don't need manually patch the code here.